src/trpc/routers/company-router/router.ts

```diff
@@ -1,7 +1,18 @@
 import { createTRPCRouter, withAuth } from "@/trpc/api/trpc";
+import { ZodOnboardingMutationSchema } from "../onboarding-router/schema";
 import { ZodSwitchCompanyMutationSchema } from "./schema";
 
 export const companyRouter = createTRPCRouter({
+  getCompany: withAuth.query(async ({ ctx }) => {
+    const user = ctx.session.user;
+
+    const company = await ctx.db.company.findFirstOrThrow({
```

Comment:
It's safe to query the user's company through membership because sometimes JWTs are stale. In case the user is removed from the company and the JWT isn't updated, this query won't fail; it would work fine.

Reply:
Hmm.. Good catch. Thanks for the heads up